core/src/components/tab-button/tab-button.tsx

```diff
@@ -37,7 +37,7 @@ export class TabButton implements ComponentInterface {
   /**
    * The URL which will be used as the `href` within this tab's button anchor.
    */
-  @Prop() href?: string;
+  @Prop() href = '#';
```

Comment:
why?

Reply:
https://stackoverflow.com/questions/41476292/how-to-give-focus-to-anchor-tag-without-href

`a` elements are not focusable without an `href`. We _can_ change that by setting `tabindex` but I don't think it's a good idea to interfere with the `tabindex` order, especially if the user wants to customize that.

Reply:
We could try `tabindex=0` on all of them and see how that, but I'm not really sure if that's going to cause issues or not.